controllers/provisioners/eks/helpers.go

```diff
@@ -457,14 +458,46 @@ func (ctx *EksInstanceGroupContext) GetLabelList() []string {
 	return labelList
 }
 
-func (ctx *EksInstanceGroupContext) GetBootstrapArgs() string {
+func (ctx *EksInstanceGroupContext) GetComputedBootstrapOptions() *v1alpha1.BootstrapOptions {
 	var (
 		instanceGroup = ctx.GetInstanceGroup()
 		configuration = instanceGroup.GetEKSConfiguration()
 	)
+	if instanceGroup.GetAnnotations()[CustomNetworkingEnabledAnnotation] == "true" {
+		hostNetworkPods, err := strconv.ParseInt(instanceGroup.GetAnnotations()[CustomNetworkingHostPodsAnnotation], 10, 64)
+		if err != nil {
+			hostNetworkPods = 2 //Default on EKS. Kube-Proxy and AWS VPC CNI
+		}
+
+		instanceTypes, _ := ctx.AwsWorker.DescribeInstanceTypes()
```

Comment:
Can you access the cache instead of making another call?

You might have to make the actual call not only in the case of launch template here:
https://github.com/keikoproj/instance-manager/blob/master/controllers/provisioners/eks/cloud.go#L233-L235
You can then cache it in DiscoveredState and use it here.

We try to avoid making additional read calls within the reconcile and rely on the state discovered in the beginning of the reconcile.

Reply:
Yep, makes sense